readthedocs/doc_builder/python_environments.py

```diff
@@ -84,13 +84,15 @@ def venv_path(self):
 
     def setup_base(self):
         site_packages = '--no-site-packages'
+        no_download = '--no-download'
         if self.config.use_system_site_packages:
             site_packages = '--system-site-packages'
         env_path = self.venv_path()
         self.build_env.run(
             self.config.python_interpreter,
             '-mvirtualenv',
             site_packages,
+            no_download,
```

Comment:
as `no_download` is not set in a conditional way, I'd say the variable can be removed and statically set on the list of arguments passed into `build_env.run`.